modules/regex/doc_classes/RegEx.xml

```diff
@@ -43,6 +43,18 @@
 		    results.push_back(result.get_string())
 		# The `results` array now contains "One", "Two", "Three".
 		[/codeblock]
+		[b]Example of wrapping a string to a maximum number of characters using RegEx:[/b]
+		[codeblock]
+		static func wrap_string(p_string: String, p_wrap_length: int) -> String:
```

Comment:
```suggestion
		static func wrap_string(p_string: String, p_wrap_length: int) -&gt; String:
```
Needs to be html escaped

Reply:
Does the reviewer typically 'resolve' the conversation, or can the PR initiator do it too after the change is force-pushed?

Reply:
Either or, I usually leave it up to the author of the PR, and resolve my own, but you are free to resolve yourself, assuming it is in fact resolved (if a reviewer thinks it is not they can unresolve to continue discussion)